config/locales/server.en.yml

```diff
@@ -2307,7 +2307,7 @@ en:
     app_association_android: "Contents of <a href='%{base_path}/.well-known/assetlinks.json'>.well-known/assetlinks.json</a> endpoint, used for Google's Digital Asset Links API."
     app_association_ios: "Contents of <a href='%{base_path}/apple-app-site-association'>apple-app-site-association</a> endpoint, used to create Universal Links between this site and iOS apps."
 
-    share_anonymized_statistics: "Share anonymized usage statistics."
+    share_anonymized_statistics: "Enable sharing of anonymized usage statistics. When this setting is enabled, data concerning the usage of the site is collected and shared in anonymized form, ensuring no personal information is disclosed."
```

Comment:
Should this setting say who the anonymized stats are shared with (either Discourse or CDCK Inc.)?

Reply:
Good point, will update to say CDCK Inc :+1: